commons/src/metadata.rs

```diff
@@ -3,11 +3,12 @@
 use serde_derive::Deserialize;
 
 /// Templated URL for release index.
-pub static RELEASES_JSON: &str =
-    "https://builds.coreos.fedoraproject.org/prod/streams/${stream}/releases.json";
+// pub static RELEASES_JSON: &str =
+//     "https://builds.coreos.fedoraproject.org/prod/streams/${stream}/releases.json";
+pub static RELEASES_JSON: &str = "http://localhost:8000/releases.json";
```

Comment:
will want to change this back before final merge.

Reply:
Whoops, this slipped by!